docs/v3/chunk-key-encoding.rst

```diff
@@ -0,0 +1,75 @@
+.. _chunk-key-encoding-list:
+
+===================
+Chunk Key Encodings
+===================
+
+The following core chunk key encodings are defined by the maintainers of
+the Zarr specification. Being listed below does not imply that a chunk key encoding is
+required to be implemented by implementations.
+
+Core chunk key encodings
+------------------------
+
+The following encodings are defined:
+
+``default``
```

Comment:
shouldn't this be in its own independent spec document, like the codecs?

Reply:
For dtypes it felt overkill. I wasn't sure here. Certainly happy to go with the separate docs if we are generally in support of the strategy.

Reply:
I think a uniform addressing scheme is good, so each separate data type should be a separate document, since presumably that's how extension data types will work.

Reply:
Since there is only very little specification to each data type, I think separate pages would be overkill.

Reply:
Pushed the `chunk-*` subdocuments.